pages/linux/gummy.md

```diff
@@ -0,0 +1,28 @@
+# gummy
+
+> Screen brightness/temperature manager for Linux/X11.
+> More information: <https://github.com/Fushko/gummy>.
+
+- Set the screen temperature to 3000K:
+
+`gummy --temperature {{3000}}`
+
+- Set the screen backlight to 50%:
+
+`gummy --backlight {{50}}`
+
+- Set the screen pixel brightness to 45%:
+
+`gummy --brightness {{45}}`
+
+- Increase current screen pixel brightness by 10%:
+
+`gummy --brightness {{+10}}`
+
+- Decrease current screen pixel brightness by 10%:
+
+`gummy --brightness {{-10}}`
+
+- Set the temperature and pixel brightness for the second screen:
+
+`gummy --screen 1 --temperature {{3800}} --brightness {{65}}`
```

Comment:
I have a doubt, is the second screen value 1?

So first screen is 0?

Reply:
Yes, the screen index is 0-based; That was the primary reason for adding that example command.

Quoting the help menu `gummy --help`:
```
-s,--screen INT:INT in [0 - 99]
```

Quoting the [project's README](https://github.com/Fushko/gummy/blob/master/README.md#usage):
> Screen settings are applied to all screens, unless the -s option with a screen number (starting from 0) is provided. All relevant settings can be increased or decreased by prefixing the value with a "+" or "-" sign.

Reply:
Thanks for the information, I will mark this as resolved.

Reply:
```suggestion
`gummy --screen {{1}} --temperature {{3800}} --brightness {{65}}`
```

So, this argument should be a placeholder.

Reply:
@einverne totally! adjusted.